ledger/participant-integration-api/src/main/scala/platform/store/backend/common/CommonStorageBackend.scala

```diff
@@ -264,12 +235,13 @@ private[backend] trait CommonStorageBackend[DB_BATCH] extends StorageBackend[DB_
       |    configuration_entries,
       |    parameters
       |  where
-      |    ledger_offset > {startExclusive} and
+      |    ({startExclusive} is null or ledger_offset>{startExclusive}) and
```

Comment:
Why this change? Can the start offset ever be null?

Reply:
i believe @nmarton-da ran into a problem where he was seeing null being passed (due to a bug) and put this in for defensiveness

Reply:
The ledger_offset is serialized from an Offset, this can be a null offset (empty array) in case we want to start the range from the very beginning, which will be an empty string, which works with h2 and postgres, but not with oracle. this special treatment takes care of it. @dasormeter I took your treatment for the mutable schema as is.

Reply:
So comparing some string with a `null` value (`ledger_offset > null`) gives different results in Oracle and Postgres?

Reply:
ah this is because an empty string is considered `null` by oracle, so we have to add the extra check for null to handle that case for empty string https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/Nulls.html#GUID-B0BA4751-9D88-426A-84AD-BCDBD5584071
```Note:Oracle Database currently treats a character value with a length of zero as null. However, this may not continue to be true in future releases, and Oracle recommends that you do not treat empty strings the same as nulls.```

Reply:
Wow, not even string comparison is standartized in SQL, and Oracle evaluates `'' is null` to `true`?

The change makes sense now.